app/services/update_existing_record_action.rb

```
@@ -0,0 +1,37 @@
+#   result   = { id: 5, name: "Alice B.", score: 80 }
+#
+# Use case:
+#   "Update existing records with imported values"
+#
+# Importer will delete the original conflicting record and replace it
+# with the merged one.
+# ===============================================================
+class UpdateExistingRecordAction
+
+  def on_duplicate_record(klass, records)
+    merged = {}
```

RuboCop (1.86.1)

[warning] Useless assignment to variable - `merged`.

(Lint/UselessAssignment)